scripts/e2e.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+git_ref=$(git rev-parse --short HEAD)
+tower_openshift_project=laura-tower-e2e-"$git_ref"
+
+echo "tower_openshift_project: '"$tower_openshift_project"'" >> e2e_test_config.yml
+echo "tower_openshift_password: '"$TOWER_OPENSHIFT_PASSWORD"'" >> e2e_test_config.yml
+echo "tower_openshift_username: '"$TOWER_OPENSHIFT_USERNAME"'" >> e2e_test_config.yml
+echo "tower_openshift_master_url: '"$OPENSHIFT_MASTER"'" >> e2e_test_config.yml
+
+
+echo "Installing tower on openshift: ${OPENSHIFT_MASTER}"
+ansible-playbook -i ./inventories/hosts \
+playbooks/install_tower.yml \
+--extra-vars "@e2e_test_config.yml"
+
+sleep 120
```

Comment:
We could add in a temporary echo statement here stating that it's waiting on all Ansible Tower Pods to be running

Reply:
Yeah sounds good. This will be a temporary fix for now. It definitely needs to be something in the ansible playbook which checks it.